main.cpp

```
@@ -1,497 +1,717 @@
+  lcd.setCursor(3, 0);
+  lcd.print(F("Heat Plates"));
+  lcd.setCursor(3, 1);
+  lcd.print(F("by Badn1nja"));
+  SETUP_PINS();
+  SPI_1.begin();
+  plate0.sensor.begin(plate0.csPin, RTD_3_WIRE, RTD_TYPE_PT100);
+  plate1.sensor.begin(plate1.csPin, RTD_3_WIRE, RTD_TYPE_PT100);
+  // Ensures sensors are ready before starting
+  do {
+    int i;
+    i++;
```

Cppcheck (2.10-2)

[error] Uninitialized variable

(uninitvar)